optaplanner-benchmark/src/build/revapi-config.json

```diff
@@ -0,0 +1,95 @@
+{
+  "revapi": {
+    "java": {
+      "filter": {
+        "packages": {
+          "comment": "We don't want to check transitive classes, since we already check them in their own module.",
+          "regex": true,
+          "include": [
+            "org\\.optaplanner\\.benchmark\\.api.*",
+            "org\\.optaplanner\\.benchmark\\.config.*"
```

Comment:
If revapi doesn't support this, we should seriously consider not including the config packages for now.